MonoGame.Framework.Native.sln

```diff
@@ -89,6 +91,21 @@ Global
 		{60D6243F-CC40-D9B5-157F-8A5B8128B70A}.Release|Any CPU.Build.0 = Release|x64
 		{60D6243F-CC40-D9B5-157F-8A5B8128B70A}.Release|x64.ActiveCfg = Release|x64
 		{60D6243F-CC40-D9B5-157F-8A5B8128B70A}.Release|x64.Build.0 = Release|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Debug_VK|Any CPU.ActiveCfg = Debug|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Debug_VK|x64.ActiveCfg = Debug|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Debug_VK|x64.Build.0 = Debug|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Debug|Any CPU.ActiveCfg = Debug|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Debug|Any CPU.Build.0 = Debug|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Debug|x64.ActiveCfg = Debug|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Debug|x64.Build.0 = Debug|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Release_VK|Any CPU.ActiveCfg = Release|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Release_VK|Any CPU.Build.0 = Release|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Release_VK|x64.ActiveCfg = Release|x64
+		{CC33DB03-389E-8F7A-81DC-4020ED856DCF}.Release_VK|x64.Build.0 = Release|x64

```

Comment:
I should probably fix this to not be `Release_VK` and `Debug_VK`.  I think i set that up when i original thought that we would have one vcproj with both Vulkan and DirectX support in it.

Thoughts?

Reply:
If they are separate project files, I think that "debug" and "release" should fit.